tests/unit_tests/framework/test_tacs_aerothermoelastic_with_solver.py

```diff
@@ -111,7 +111,7 @@ def test_coupled_derivatives(self):
         # Check whether to use the complex-step method or now
         complex_step = False
         epsilon = 1e-5
-        rtol = 1e-4
+        rtol = 1e-3
```

Comment:
Relative tolerances should hopefully be able to be uniform across different tests.